tests/php/tests/includes/test_class.wp-job-manager-usage-tracking-data.php

```diff
@@ -53,6 +53,20 @@ class WP_Test_WP_Job_Manager_Usage_Tracking_Data extends WPJM_BaseTest {
 	 */
 	private $publish;
 
+	/**
+	 * Set up for tests.
+	 */
+	public function setUp() {
```

Comment:
If this only needs to run once for this entire test file, instead of once per test case, then a better place for this code would be in the static `setUpBeforeClass` function. This will save us some build time.